src/Components/settings-dropdown/setting-index.tsx

```diff
@@ -0,0 +1,115 @@
+const ORG_URL = "http://localhost:8080/organizations";
+const USR_URL = "http://localhost:8080/users";
+
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import "./settings.css";
+
+const SettingsIndex: React.FC = () => {
+  const { id } = useParams<{ id: string }>(); // Get the organization ID from the route params
+  const [emailSettings, setEmailSettings] = useState<any>({});
+  const [organizationName, setOrganizationName] = useState<string>("");
+  const [adminEmail, setAdminEmail] = useState<string>("");
+
+  useEffect(() => {
+    // Fetch email settings from the API
+    const fetchEmailSettings = async () => {
+      try {
+        const response = await fetch(
+          // `${ORG_URL}/${id}/email-settings`
+          `${ORG_URL}/669f03102a97b116272c2085/email-settings` // Hardcoded organization ID for now
```

Comment:
**Actionable**
Please prefix the comment with a @todo here. (@todo's are easy to find, there are even plugins to tell you about all your todo's in your codebase)